app/assets/javascripts/collections/story_collection.js

```diff
@@ -18,25 +18,62 @@ module.exports = Backbone.Collection.extend({
     this.labels = [];
   },
 
+  saveSorting: function(columnName) {
+    var column = this;
+    if(columnName) {
+      column = this.column(columnName);
+    }
+    var orderedIds = column.map(
+      function(model) {
+        return model.id;
+      }
+    );
+    Backbone.ajax({
+      method: 'PUT',
+      url: this.url + '/sort',
+      data: { ordered_ids: orderedIds }
+    });
+  },
+
   comparator: function(story) {
     return story.position();
   },
 
-  next: function(story) {
-    var index = this.indexOf(story) + 1;
-    if(index >= this.length) {
+  columnStoryIndex: function(story) {
```

Comment:
I think you are missing a verb in a couple of function names. 
IMO, using verbs in function names helps improving readability and makes it easier to know what is the purpose of that function.

Example: 
`getColumnStoryIndex` instead of `columnStoryIndex`
`storyByIndexOnColumn` instead of `getStoryByIndexOnColumn`